api/mapping/views.py

```diff
@@ -1152,6 +1152,13 @@ def form_valid(self, form):
         ):
             return self.form_invalid(form)
 
+        # Form is invalid if visibility is restricted and no viewers are set
```

Comment:
I don't think we need this check - it's perfectly valid to have a RESTRICTED SR that just has an admin and editor. They both inherit permissions to view, so we don't technically need any viewers.